src/single-spa-preact.js

```diff
@@ -2,6 +2,8 @@ const defaultOpts = {
 	// required opts
 	preact: null,
 	rootComponent: null,
+	
+  // optional opts
```

Comment:
This file has a mixture of tabs and spaces. It was originally tabs - I'm okay with changing to spaces but we should only have one kind of indentation.

Reply:
Oof. Sorry. Not sure what happened when I pasted. Should be good now.

Reply:
Oof. Sorry. Not sure what happened when I pasted. Should be good now.